snuba/web/query.py

```diff
@@ -73,7 +72,9 @@ def _run_query_pipeline(
     - Providing the newly built Query, processors to be run for each DB query and a QueryRunner
       to the QueryExecutionStrategy to actually run the DB Query.
     """
-    entity = get_entity(EntityKey(request.query.get_entity_name()))
+    query_entity = request.query.get_entity()
+    assert query_entity is not None
```

Comment:
This assert is already done in the `get_entity` function.

Reply:
you are right, thanks, will fix